## 市のオープンデータの緯度経度取得スクリプト作成 #7

1. 対象ウェブサイトが施設カテゴリ毎に下記の構造を持っていたため、loopを回してテーブル情報を取得
    - 保育園: https://www.city.funabashi.lg.jp/funakkonavi/map/0/index.html
    - 幼稚園: https://www.city.funabashi.lg.jp/funakkonavi/map/1/index.html
    - ...
    - その他預け先: https://www.city.funabashi.lg.jp/funakkonavi/map/12/index.html
    
2. それぞれのテーブルの住所に対してgoogle geocode APIを利用して経度・緯度を取得

In [31]:
from bs4 import BeautifulSoup

url=f"https://www.city.funabashi.lg.jp/funakkonavi/map/0/index.html"
context = ssl._create_unverified_context()
response = request.urlopen(url, context=context)
html = response.read()

soup = BeautifulSoup(html, 'html.parser')

# 施設カテゴリ
res = soup.select("select#category-map")

# [<option selected="selected" value="">選択してください</option>,を除く
results = res[0].find_all()

id2CategoryType = {idx:t.text for idx, t in enumerate(results[1:])}

In [92]:
id2CategoryType

{0: '保育園',
 1: '幼稚園',
 2: '認定こども園',
 3: '小規模保育事業所',
 4: '家庭的保育者',
 5: '事業所内保育事業所',
 6: '認証保育所',
 7: '認可外保育施設',
 8: '一時保育',
 9: '病児・病後児保育',
 10: '児童ホーム',
 11: '子育て支援センター',
 12: 'その他預け先'}

In [33]:
import pandas as pd
from urllib import request
import ssl
# , flavor="bs4"
# private PCジャないからか、SSL errorが発生する。定期実行をする際はよくないけど下記のやり方でスクレーピングする。

all_facilities = {}
for i in range(13):
    url=f"https://www.city.funabashi.lg.jp/funakkonavi/map/{i}/index.html"
    context = ssl._create_unverified_context()
    response = request.urlopen(url, context=context)
    html = response.read()
    category = id2CategoryType[i]
    print(category)
    try:
        facilities = pd.read_html(html, )
    except ValueError as e:
        print(f"ValueError: No tables found for https://www.city.funabashi.lg.jp/funakkonavi/map/{i}/index.html")
        facilities = []
    all_facilities[category] = facilities


保育園
幼稚園
認定こども園
小規模保育事業所
家庭的保育者
事業所内保育事業所
ValueError: No tables found for https://www.city.funabashi.lg.jp/funakkonavi/map/5/index.html
認証保育所
認可外保育施設
一時保育
病児・病後児保育
児童ホーム
子育て支援センター
その他預け先
ValueError: No tables found for https://www.city.funabashi.lg.jp/funakkonavi/map/12/index.html


In [35]:
all_facilities["児童ホーム"]

[          施設名                          住所      電話番号(代表)
 0     前原児童ホーム    〒274-0825千葉県船橋市前原西6-1-12  047-475-5454
 1    高根台児童ホーム     〒274-0065千葉県船橋市高根台2-2-2  047-466-3638
 2   習志野台児童ホーム    〒274-0063千葉県船橋市習志野台3-4-1  047-464-0001
 3    金杉台児童ホーム     〒273-0852千葉県船橋市金杉台2-2-4  047-447-7837
 4     若松児童ホーム      〒273-0013千葉県船橋市若松2-3-6  047-433-7991
 5     西船児童ホーム       〒273-0033千葉県船橋市本郷町554  047-333-2007
 6     小室児童ホーム      〒270-1471千葉県船橋市小室町3308  047-457-1000
 7     三山児童ホーム     〒274-0072千葉県船橋市三山2-42-3  047-478-6800
 8   八木が谷児童ホーム    〒274-0804千葉県船橋市みやぎ台1-7-1  047-448-4400
 9    松が丘児童ホーム   〒274-0064千葉県船橋市松が丘1-52-22  047-463-5087
 10   飯山満児童ホーム  〒274-0822千葉県船橋市飯山満町2-488-8  047-468-0411
 11    夏見児童ホーム    〒273-0865千葉県船橋市夏見4-39-15  047-423-5999
 12    塚田児童ホーム    〒273-0042千葉県船橋市前貝塚町601-1  047-439-9299
 13    宮本児童ホーム     〒273-0003千葉県船橋市宮本6-18-1  047-424-9842
 14    三咲児童ホーム     〒274-0812千葉県船橋市三咲3-5-10  047-448-2397
 15   新高根児童ホーム    〒274-0814千葉県船橋市新高根1-12-9  047-469-3940
 16   薬円台児童ホーム    〒274-0077千葉県船

In [42]:
# !pip install pygeocoder
# !pip install googlemaps

In [43]:
import googlemaps

In [46]:
googleapikey = os.environ["GOOGLE_API_KEY"] 

In [47]:
import pandas as pd
import urllib
import urllib.error
import urllib.request

# Google API モジュール
from pygeocoder import Geocoder
import googlemaps

In [54]:
gmaps = googlemaps.Client(key=googleapikey)

In [55]:
# gmaps.geocode(address)

In [58]:
r = all_facilities["児童ホーム"][0]["住所"].map(lambda loc: gmaps.geocode(loc))

In [80]:
# r[0][0]['address_components']
# # len(r[0][0])
# r[0][0].keys()

In [75]:
# こんな感じでデータ収集完了
r[0][0]['geometry']["viewport"]

{'northeast': {'lat': 35.7040551802915, 'lng': 140.0267875802915},
 'southwest': {'lat': 35.70135721970851, 'lng': 140.0240896197085}}

In [78]:
all_facilities["児童ホーム"][0]["geocodes"] = r.map(lambda x: x[0]['geometry']["viewport"], na_action="ignore")

In [89]:
# all_facilities["児童ホーム"][0]

In [ ]:
from IPython.display import display

In [90]:
for k in all_facilities.keys():
    print(k)
    print(len(all_facilities[k]))
    if len(all_facilities[k]) == 0:
        continue
    r = all_facilities[k][0]["住所"].map(lambda loc: gmaps.geocode(loc))
    all_facilities[k][0]["geocodes"] = r.map(lambda x: x[0]['geometry']["viewport"], na_action="ignore")
    display(all_facilities[k][0])

保育園
1


施設名                        住所  \
0              宮本第一保育園  〒273-0003千葉県船橋市宮本6-26-27   
1              宮本第二保育園   〒273-0003千葉県船橋市宮本8-9-10   
2                若松保育園    〒273-0013千葉県船橋市若松2-6-3   
3                湊町保育園  〒273-0011千葉県船橋市湊町1-16-23   
4                千鳥保育園    〒273-0004千葉県船橋市南本町13-1   
..                 ...                       ...   
118    ミアヘルサ保育園ひびき新船橋北     〒273-0045　船橋市山手1-1-11   
119         船橋馬込沢雲母保育園   〒273-0046　船橋市上山町2-487-1   
120         船橋こどもの木保育園    〒274-0815　船橋市西習志野2-6-1   
121  京進のほいくえん HOPPA前原西   〒274-0825　船橋市前原西4-12-15   
122             ここみ保育園      〒273-0021　船橋市海神1-8-9   

                             電話番号(代表)  \
0                        047-442-5639   
1                        047-423-1304   
2                        047-433-4041   
3                        047-431-4961   
4                        047-431-5061   
..                                ...   
118  電話：047-498-9370 FAX：047-498-9371   
119  電話：047-430-2424 FAX：047-430-2425   
120  電話：047-468-8207 FAX：047-468-8208   
121  電話：047-489-5260 FAX：047-489-5261   
122  電話：047-432-2904 FAX：047-432-2904   

                                              geocodes  
0    {'northeast': {'lat': 35.6992823802915, 'lng':...  
1    {'northeast': {'lat': 35.6928209802915, 'lng':...  
2    {'northeast': {'lat': 35.6808319302915, 'lng':...  
3    {'northeast': {'lat': 35.6941728802915, 'lng':...  
4    {'northeast': {'lat': 35.69943578029149, 'lng'...  
..                                                 ...  
118  {'northeast': {'lat': 35.7161008, 'lng': 139.9...  
119  {'northeast': {'lat': 35.73759818029149, 'lng'...  
120  {'northeast': {'lat': 35.7268150802915, 'lng':...  
121  {'northeast': {'lat': 35.6997823802915, 'lng':...  
122  {'northeast': {'lat': 35.70168038029149, 'lng'...  

[123 rows x 4 columns]

幼稚園
1


施設名                            住所      電話番号(代表)  \
0         恵楓幼稚園     〒273-0003　千葉県船橋市宮本8-35-12  047-424-5100   
1         若松幼稚園       〒273-0013　千葉県船橋市若松2-3-3  047-431-7751   
2         船橋幼稚園     〒273-0005　千葉県船橋市本町3-36-32  047-424-3351   
3       ひなぎく幼稚園     〒273-0021　千葉県船橋市海神4-29-10  047-431-3569   
4         二葉幼稚園      〒273-0021　千葉県船橋市海神5-7-14  047-431-4701   
5         海神幼稚園     〒273-0021　千葉県船橋市海神6-18-34  047-431-5708   
6         山野幼稚園      〒273-0031　千葉県船橋市西船2-10-7  047-431-2431   
7         西船幼稚園     〒273-0031　千葉県船橋市西船2-32-25  047-431-2932   
8         葛飾幼稚園      〒273-0031　千葉県船橋市西船4-16-4  047-431-4463   
9      船橋ひかり幼稚園        〒273-0034　千葉県船橋市二子町498  047-335-5998   
10      健伸行田幼稚園       〒273-0044　千葉県船橋市行田2-4-3  047-439-2125   
11        清和幼稚園       〒273-0042　千葉県船橋市前貝塚町645  047-439-1798   
12      コスモス幼稚園      〒273-0046　千葉県船橋市上山町3-512  047-438-2790   
13        健伸幼稚園      〒273-0048　千葉県船橋市丸山5-12-7  047-438-7019   
14       日の丸幼稚園    〒273-0855　千葉県船橋市馬込西2-17-11  047-438-4545   
15    船橋ひまわり幼稚園     〒273-0865　千葉県船橋市夏見5-26-22  047-422-7051   
16       シオン幼稚園       〒273-0865　千葉県船橋市夏見6-6-6  047-422-5644   
17       夏見台幼稚園     〒273-0866　千葉県船橋市夏見台2-16-1  047-429-7231   
18        金杉幼稚園       〒273-0853　千葉県船橋市金杉3-6-1  047-439-2525   
19  第二船橋ひまわり幼稚園      〒273-0853　千葉県船橋市金杉8-4-12  047-448-3960   
20      みどり台幼稚園         〒274-0818　千葉県船橋市緑台2-6  047-448-3311   
21     みのり第二幼稚園      〒274-0816　千葉県船橋市芝山2-5-17  047-467-6059   
22        栄光幼稚園      〒274-0816　千葉県船橋市芝山3-10-9  047-464-8084   
23       冨士見幼稚園      〒274-0065　千葉県船橋市高根台1-1-1  047-465-2555   
24       木戸脇幼稚園     〒274-0065　千葉県船橋市高根台7-17-3  047-465-6401   
25        神明幼稚園     〒274-0822　千葉県船橋市飯山満町1-640  047-422-3447   
26       飯山満幼稚園  〒274-0822　千葉県船橋市飯山満町3-1716-2  047-467-1929   
27     冨士見第二幼稚園     〒274-0825　千葉県船橋市前原西6-1-13  047-474-0324   
28        中台幼稚園    〒274-0824　千葉県船橋市前原東4-16-11  047-472-7381   
29      たきのい幼稚園    〒274-0073　千葉県船橋市田喜野井6-20-3  047-463-1986   
30      すずらん幼稚園      〒274-0072　千葉県船橋市三山3-40-8  047-472-8726   
31      船橋小鳩幼稚園      〒274-0072　千葉県船橋市三山5-34-3  047-473-3734   
32      習志野台幼稚園   〒274-0063　千葉県船橋市習志野台2-59-22  047-466-2151   
33        瑞穂幼稚園   〒274-0063　千葉県船橋市習志野台4-42-20  047-465-3300   
34     船橋いづみ幼稚園   〒274-0063　千葉県船橋市習志野台6-11-22  047-466-5082   
35       木の実幼稚園      〒274-0060　千葉県船橋市坪井東4-7-1  047-457-2055   
36       古和釜幼稚園     〒274-0064　千葉県船橋市松が丘4-32-1  047-464-3030   
37       すずみ幼稚園     〒274-0067　千葉県船橋市大穴南3-8-12  047-464-7121   
38        英進幼稚園   〒274-0804　千葉県船橋市みやぎ台3-10-10  047-448-7066   
39      八木ケ谷幼稚園    〒274-0802　千葉県船橋市八木が谷2-18-1  047-447-6729   
40        坪井幼稚園      〒274-0062　千葉県船橋市坪井町133-1  047-456-1075   

                                             geocodes  
0   {'northeast': {'lat': 35.6922564802915, 'lng':...  
1   {'northeast': {'lat': 35.6799572802915, 'lng':...  
2   {'northeast': {'lat': 35.6983850302915, 'lng':...  
3   {'northeast': {'lat': 35.71262428029149, 'lng'...  
4   {'northeast': {'lat': 35.7085295802915, 'lng':...  
5   {'northeast': {'lat': 35.7066712802915, 'lng':...  
6   {'northeast': {'lat': 35.71180248029149, 'lng'...  
7   {'northeast': {'lat': 35.7178237802915, 'lng':...  
8   {'northeast': {'lat': 35.7118348802915, 'lng':...  
9   {'northeast': {'lat': 35.7123739802915, 'lng':...  
10  {'northeast': {'lat': 35.7194299802915, 'lng':...  
11  {'northeast': {'lat': 35.7273113802915, 'lng':...  
12  {'northeast': {'lat': 35.7359040802915, 'lng':...  
13  {'northeast': {'lat': 35.74771468029149, 'lng'...  
14  {'northeast': {'lat': 35.7411847802915, 'lng':...  
15  {'northeast': {'lat': 35.7122926802915, 'lng':...  
16  {'northeast': {'lat': 35.7165935, 'lng': 139.9...  
17  {'northeast': {'lat': 35.7227269802915, 'lng':...  
18  {'northeast': {'lat': 35.73269698029149, 'lng'...  
19  {'northeast': {'lat': 35.73333128029149, 'lng'...  
20  {'northeast': {'lat': 35.73022168029149, 'lng'...  
21  {'northeast': {'lat': 35.7213759802915, 'lng':...  
22  {'northeast': {'lat': 35.7177351802915, 'lng':...  
23  {'northeast': {'l

認定こども園
1


施設名                        住所  \
0               船橋旭こども園    〒273-0866　船橋市夏見台5-7-13   
1             田喜野井旭こども園  〒274-0073　船橋市田喜野井4-30-12   
2          認定こども園 不二幼稚園    〒274-0822　船橋市飯山満町2-666   
3            認定こども園おひさま    〒274-0825　船橋市前原西6-1-64   
4     幼保連携型認定こども園 大浜幼稚園     〒274-0077　船橋市薬円台4-6-3   
5         そらまめこども園 船橋駅前     〒273-0005　船橋市本町1-30-1   
6             高根台文化こども園     〒274-0065　船橋市高根台3-3-2   
7  幼保連携型認定こども園　二和ひつじ幼稚園   〒274-0805　船橋市二和東6-23-21   

                            電話番号(代表)  \
0   電話：047-439-0010 FAX：047-430-0238   
1   電話：047-456-3180 FAX：047-461-2000   
2  電話：047-466-2925 FAX ：047-466-2989   
3   電話：047-476-7802 FAX：047-411-7705   
4  電話：047-466-5720 FAX ：047-466-5954   
5   電話：047-405-9951 FAX：047-405-9952   
6   電話：047-466-2511 FAX：047-466-2511   
7   電話：047-447-2041 FAX：047-447-2041   

                                            geocodes  
0  {'northeast': {'lat': 35.73188278029149, 'lng'...  
1  {'northeast': {'lat': 35.7046346802915, 'lng':...  
2  {'northeast': {'lat': 35.71241698029149, 'lng'...  
3  {'northeast': {'lat': 35.7042374802915, 'lng':...  
4  {'northeast': {'lat': 35.7147413802915, 'lng':...  
5  {'northeast': {'lat': 35.70173928029149, 'lng'...  
6  {'northeast': {'lat': 35.7329678802915, 'lng':...  
7  {'northeast': {'lat': 35.75638238029151, 'lng'...

小規模保育事業所
1


施設名                                    住所  \
0        小規模保育所 リトルキディ  〒274-0060　船橋市坪井東3-10-1　グランソシア船橋芽吹の杜内   
1             うみのほいくえん  〒273-0011　船橋市湊町2-2-20　アソルティ船橋102・202   
2            パルパステル保育園      〒273-0005　船橋市本町5-14-15　パルパステル101   
3             キッズランド未来        〒274-0805　船橋市二和東6-18-27　山和ビル1F   
4         保育ルーム キューティー                〒274-0824　船橋市前原東5-18-4   
5         太陽の子 船橋三咲保育園            〒274-0813　船橋市南三咲1-20-10　1F   
6            ひだまり滝台保育園                〒274-0074　船橋市滝台1-12-12   
7   京進のほいくえん HOPPA船橋駅園                 〒273-0005　船橋市本町4-7-17   
8   京進のほいくえん HOPPA津田沼園          〒274-0825　船橋市前原西2-7-9　三真ビル2F   
9               リトル・清和                〒273-0042　船橋市前貝塚町646-8   
10          トレポンテ駅前保育園                  〒273-0005　船橋市本町6-8-1   
11      りりぱっとナーサリー津田沼園               〒274-0825　船橋市前原西2-44-10   
12       りりぱっとナーサリー中山園               〒273-0035　船橋市本中山2-23-16   
13      ココファン・ナーサリー西船橋                   〒273-0034　船橋市二子町610   
14           健伸むぎの子保育園                〒273-0048　船橋市丸山5-15-15   
15             すずらん保育園                 〒274-0072　船橋市三山3-39-2   
16        クレヨンキッズ北習志野園              〒274-0063　船橋市習志野台1-34-25   
17            東船橋色葉保育園                〒273-0002　船橋市東船橋4-31-9   
18         リップル保育園 西船橋               〒273-0032　船橋市葛飾町2-378-7   
19              ひかり保育園                〒274-0063　船橋市習志野台4-3-9   
20    ミアヘルサ保育園ひびき新船橋第二                 〒273-0045　船橋市山手1-3-17   
21         やきがやなかよし保育園                         船橋市八木が谷2-18-1   
22           サンヴェルデ保育園                           船橋市緑台2-4-17   
23             ひだまり保育園                    船橋市薬円台6-1-1　薬園台駅2階   
24             こりんご保育園         〒273-0002　船橋市東船橋1-38-3　清水ビル2階   
25       保育ルーム　サンライズ船橋         〒273-0005　船橋市本町1-25-18　ジーリオ船橋   

                            電話番号(代表)  \
0   電話：047-436-8011 FAX：047-436-8012   
1   電話：047-468-8698 FAX：047-468-8698   
2   電話：047-460-6666 FAX：047-466-2989   
3   電話：047-448-7735 FAX：047-401-9395   
4   電話：047-427-7643 FAX：047-427-7643   
5   電話：047-440-6585 FAX：047-440-6586   
6   電話：047-436-8840 FAX：047-436-8840   
7   電話：047-405-2613 FAX：047-405-2614   
8   電話：047-409-9571 FAX：047-409-9572   
9   電話：047-404-7516 FAX：047-439-1793   
10  電話：047-409-8816 FAX：047-421-0023   
11  電話：047-405-2331 FAX：047-405-2433   
12  電話：047-303-3038 FAX：047-303-3038   
13  電話：047-302-7761 FAX：047-333-3131   
14  電話：047-404-8746 FAX：047-404-8747   
15  電話：047-409-9541 FAX：047-409-9545   
16  電話：047-456-4440 FAX：047-456-4441   
17  電話：047-460-2727 FAX：047-460-2737   
18  電話：047-401-5567 FAX：047-401-5568   
19  電話：047-456-6370 FAX：047-456-6370   
20  電話：047-409-9055 FAX：047-409-9056   
21  電話：047-498-9367 FAX：047-498-9368   
22  電話：047-402-2270 FAX：047-402-2271   
23  電話：047-404-8511 FAX：047-404-8512   
24  電話：047-406-4380 FAX：047-413-0666   
25  電話：047-422-0805 FAX：047-422-0805   

                                             geocodes  
0   {'northeast': {'lat': 35.7296705802915, 'lng':...  
1   {'northeast': {'lat': 35.6971234802915, 'lng':...  
2   {'northeast': {'lat': 35.7032507802915, 'lng':...  
3   {'northeast': {'lat': 35.7572449302915, 'lng':...  
4   {'northeast': {'lat': 35.7020278802915, 'lng':...  
5   {'northeast': {'lat': 35.73710483029151, 'lng'...  
6   {'northeast': {'lat': 35.7090584302915, 'lng':...  
7   {'northeast': {'lat': 35.7023790802915, 'lng':...  
8   {'northeast': {'lat': 35.6940283802915, 'lng':...  
9   {'northeast': {'lat': 35.7276123802915, 'lng':...  
10  {'northeast': {'lat': 35.7067077802915, 'lng':...  
11  {'northeast': {'lat': 35.6955296802915, 'lng':...  
12  {'northeast': {'lat': 35.7165167802915, 'lng':...  
13  {'northeast': {'lat': 35.7100761802915, 'lng':...  
14  {'northeast': {'lat': 35.7471879802915, 'lng':...  
15  {'northeast': {'lat': 35.6984836802915, 'lng':...  
16  {'northeast': {'lat': 35.72445648029149, 'lng'...  
17  {'northeast': {'lat': 35.70086798029149, 'lng'...  
18  {'northeast': {'lat': 35.7084629802915, 'lng':...  
19  {'northeast': {'lat': 35.7167684802915, 'lng':...  
20  {'northeast': {'lat': 35.7128

家庭的保育者
1


施設名                    住所  \
0                            清水    〒273-0045　船橋市山手2丁目   
1                             林    〒274-0072　船橋市三山4丁目   
2                            飯田  〒274-0073　船橋市田喜野井2丁目   
3                            桑原   〒274-0064　船橋市松が丘5丁目   
4  片渕（令和2年4月1日～令和3年3月31日まで事業休止）   〒274-0814　船橋市新高根1丁目   

                                            geocodes  
0  {'northeast': {'lat': 35.7175555, 'lng': 139.9...  
1  {'northeast': {'lat': 35.7022897, 'lng': 140.0...  
2  {'northeast': {'lat': 35.7006683, 'lng': 140.0...  
3  {'northeast': {'lat': 35.7375105, 'lng': 140.0...  
4  {'northeast': {'lat': 35.7278249, 'lng': 140.0...

事業所内保育事業所
0
認証保育所
1


施設名  \
0                             グリーンベア・ベイサイド   
1  保育ステーション　ハミングバード　※令和3年4月1日 小規模保育事業へ移行予定   
2               キッズサポート　かむ　※令和3年3月31日 廃止予定   

                                           住所      電話番号(代表)  \
0                     〒273-0005千葉県船橋市本町6-8-36  047-460-1255   
1  〒274-0063千葉県船橋市習志野台3-1-1 EKITA KITANARA 3F  047-401-4795   
2              〒273-0003　船橋市宮本2-9-1 第二ときわビル2階  047-425-4300   

                                            geocodes  
0  {'northeast': {'lat': 35.7066708302915, 'lng':...  
1  {'northeast': {'lat': 35.72264098029149, 'lng'...  
2  {'northeast': {'lat': 35.69606178029151, 'lng'...

認可外保育施設
1


施設名  \
0                                    キッズハウスくれよん   
1                                        あずかルンバ   
2                          リトルガーデンインターナショナルスクール   
3                                 西船橋北口駅前キッズルーム   
4                                   駅前保育園　りりぱっと   
5                   E-Life International School   
6                          船橋インターナショナルスクール[FIS]   
7                             Kids Club カリン（休園）   
8                           ICEモンテッソーリこどものいえ西船橋   
9                                 キッズルーム　森のくまさん   
10                              ファミリー保育室分園　ふれんど   
11                             グリーンベアベイサイド船橋夏見校   
12                                    キッズステーション   
13                                   ナーサリーホーム船橋   
14                             カデナ・アメリカン・プリスクール   
15                                   りりぱっとsteps   
16                                         菅長学園   
17                                  ひまわりガーデン西船橋   
18  Milky Way International Nursery Center下総中山校   
19                                    RuRi船橋保育園   
20                                  コンパス幼保園　船橋校   
21                                  ひまわりガーデン薬円台   

                                         住所       電話番号(代表)  \
0                 〒273-0864千葉県船橋市北本町2-21-18   047-423-5729   
1          〒273-0012千葉県船橋市浜町2-2-7　ビビット南船橋4Ｆ   047-402-6613   
2         〒273-0012千葉県船橋市浜町2-1-1　三井ビルディング1F   047-404-4321   
3   〒273-0031千葉県船橋市西船4-21-1　デイジイファーストビル502号   047-435-6900   
4    〒274-0825千葉県船橋市前原西2-14-1　ダイアパレス津田沼1302   047-476-3318   
5        〒273-0047千葉県船橋市藤原2-18-30　ソレアードⅠ105   047-375-8866   
6     〒273-0005千葉県船橋市本町6-5-8　ステイタスYUHARA102  050-5586-2086   
7   〒273-0024千葉県船橋市海神町南1-741-1 アーティザン西船橋401   047-433-0640   
8         〒273-0032千葉県船橋市葛飾町2-336-1 キュービル5F   047-401-4765   
9          〒274-0812千葉県船橋市三咲3-1-14　増田ハイツ106  080-4435-5952   
10             〒274-0822千葉県船橋市飯山満町3-1513-2階   047-769-4619   
11                  〒273-0865千葉県船橋市夏見1-9-27   047-411-0555   
12              〒273-0044千葉県船橋市行田3-2-13-103   047-779-3802   
13                 〒273-0864千葉県船橋市北本町2-65-1   047-411-5178   
14             〒274-0825　千葉県船橋市前原西2-32-5　3階   047-474-0808   
15                 〒274-0825千葉県船橋市前原西2-11-6   047-411-8817   
16                 〒274-0072千葉県船橋市三山6-18-10   047-477-7799   
17              〒273-0024千葉県船橋市海神町南1-1742-1   047-401-8721   
18               〒273-0035千葉県船橋市本中山2丁目15番7号   047-314-8433   
19              〒274-0822　千葉県船橋市飯山満町2-627-2   047-440-8744   
20     〒273-0047千葉県船橋市藤原7-17-28　日本都市馬込沢ビル4F   047-401-0000   
21                  〒274-0077千葉県船橋市薬円台5-4-8   047-402-6752   

                                             geocodes  
0   {'northeast': {'lat': 35.7173542802915, 'lng':...  
1   {'northeast': {'lat': 35.6888507802915, 'lng':...  
2   {'northeast': {'lat': 35.68615208029149, 'lng'...  
3   {'northeast': {'lat': 35.7105324302915, 'lng':...  
4   {'northeast': {'lat': 35.6933466802915, 'lng':...  
5   {'northeast': {'lat': 35.73284698029151, 'lng'...  
6   {'northeast': {'lat': 35.7060489802915, 'lng':...  
7   {'northeast': {'lat': 35.7033325302915, 'lng':...  
8   {'northeast': {'lat': 35.7085701302915, 'lng':...  
9   {'northeast': {'lat': 35.7518763802915, 'lng':...  
10  {'northeast': {'lat': 35.7137610802915, 'lng':...  
11  {'northeast': {'lat': 35.7079877802915, 'lng':...  
12  {'northeast': {'lat': 35.7203878802915, 'lng':...  
13  {'northeast': {'lat': 35.71592468029149, 'lng'...  
14  {'northeast': {'lat': 35.6956685302915, 'lng':...  
15  {'northeast': {'lat': 35.6948322802915, 'lng':...  
16  {'northeast': {'lat': 35.7060324802915, 'lng':...  
17  {'northeast': {'lat': 35.70406378029149, 'lng'...  
18  {'northeast': {'lat': 35.71643968029149, 'lng'...  
19  {'northeast': {'lat': 35.7079285802915, 'lng':...  
20  {'northeast': {'lat': 35.74376588029149, 'lng'...  
21  {'northeast': {'lat': 35.7110213802915, 'lng':...

一時保育
1


施設名                           住所  \
0              若松幼稚園          〒273-0013船橋市若松2-3-3   
1   まなびの森保育園　船橋競馬場駅前         〒273-0003船橋市宮本9-1-11   
2              弥生保育園     〒273-0035千葉県船橋市本中山3-4-18   
3             みそら保育園     〒273-0036千葉県船橋市東中山2-1-20   
4             さくら保育園        〒273-0034千葉県船橋市二子町474   
5         アリスなかよし保育園     〒273-0031千葉県船橋市西船2-29-28   
6              湊町保育園     〒273-0011千葉県船橋市湊町1-16-23   
7          前原ひまわり保育園    〒274-0825千葉県船橋市前原西4-18-19   
8          田喜野井旭こども園   〒274-0073千葉県船橋市田喜野井4-30-12   
9           三山つくし保育園      〒274-0072千葉県船橋市三山9-31-7   
10       アンデルセン第二保育園      〒274-0063　船橋市習志野台5-43-3   
11            ひばり保育園     〒274-0064千葉県船橋市松が丘1-33-4   
12            まこと保育園     〒274-0056千葉県船橋市神保町275-34   
13           あすなろ保育園      〒274-0806千葉県船橋市二和西2-6-1   
14        ククルなかよし保育園     〒273-0865千葉県船橋市夏見2-11-43   
15           船橋旭こども園     〒273-0866千葉県船橋市夏見台5-7-13   
16       ローゼンかみやま保育園    〒273-0046千葉県船橋市上山町2-288-1   
17           シーガル保育園  〒274-0822千葉県船橋市飯山満町2-1007-1   
18          ゆいまーる保育園     〒273-0864千葉県船橋市北本町1-12-8   
19            丸山旭保育園     〒273-0048千葉県船橋市丸山5-33-25   
20          あまねの杜保育園      〒273-0044千葉県船橋市行田2-9-10   
21         東船橋ちとせ保育園    〒273-0002千葉県船橋市東船橋1-16-25   
22            古和釜幼稚園        〒274-0064船橋市松が丘4-32-1   
23             健伸幼稚園         〒273-0048船橋市丸山5-12-7   
24           八木ケ谷幼稚園       〒274-0802船橋市八木が谷2-18-1   
25             英進幼稚園      〒274-0804船橋市みやぎ台3-10-10   

                          電話番号(代表)  \
0                     047-431-7751   
1                     047-402-4402   
2   047-332-7283（一時預かり専用）※予約システム対応   
3                     047-302-2505   
4   047-302-8302（一時預かり専用）※予約システム対応   
5   047-436-1215（一時預かり専用）※予約システム対応   
6            047-431-5085（一時預かり専用）   
7                     047-474-7788   
8   047-456-3190（一時預かり専用）※予約システム対応   
9                     047-473-4435   
10  047-468-4673（一時預かり専用）※予約システム対応   
11                    047-466-5915   
12                    047-457-1685   
13                    047-447-0023   
14  047-424-0301（一時預かり専用）※予約システム対応   
15  047-439-5700（一時預かり専用）※予約システム対応   
16           047-401-6291（一時預かり専用）   
17  047-404-8600（一時預かり専用）※予約システム対応   
18  047-424-7723（一時預かり専用）※予約システム対応   
19  047-438-3385（一時預かり専用）※予約システム対応   
20                    047-401-5011   
21                    047-423-5717   
22                    047-464-3030   
23           047-438-7018（一時預かり専用）   
24                    047-447-6729   
25           047-448-2121（一時預かり専用）   

                                             geocodes  
0   {'northeast': {'lat': 35.6799572802915, 'lng':...  
1   {'northeast': {'lat': 35.69092748029149, 'lng'...  
2   {'northeast': {'lat': 35.7123291802915, 'lng':...  
3   {'northeast': {'lat': 35.7157758802915, 'lng':...  
4   {'northeast': {'lat': 35.71101528029149, 'lng'...  
5   {'northeast': {'lat': 35.7152334802915, 'lng':...  
6   {'northeast': {'lat': 35.6941728802915, 'lng':...  
7   {'northeast': {'lat': 35.69807508029151, 'lng'...  
8   {'northeast': {'lat': 35.7046346802915, 'lng':...  
9   {'northeast': {'lat': 35.6947742802915, 'lng':...  
10  {'northeast': {'lat': 35.7202151802915, 'lng':...  
11  {'northeast': {'lat': 35.7367117802915, 'lng':...  
12  {'northeast': {'lat': 35.76638003029149, 'lng'...  
13  {'northeast': {'lat': 35.74332658029149, 'lng'...  
14  {'northeast': {'lat': 35.7104087802915, 'lng':...  
15  {'northeast': {'lat': 35.73188278029149, 'lng'...  
16  {'northeast': {'lat': 35.7339109802915, 'lng':...  
17  {'northeast': {'lat': 35.71480298029149, 'lng'...  
18  {'northeast': {'lat': 35.71426888029149, 'lng'...  
19  {'northeast': {'lat': 35.7444570802915, 'lng':...  
20  {'northeast': {'lat': 35.72412618029149, 'lng'...  
21  {'northeast': {'lat': 35.7035542, 'lng': 140.0...  
22  {'northeast': {'lat': 35.73417218029149, 'lng'...  
23  {'northeast': {'lat': 35.74771468029149, 'lng'...  
24  {'northeast': {'lat': 35.7620307802915, 'lng':...  
25  {'northeast': {'lat': 35.75582728029149, 'lng'...

病児・病後児保育
1


施設名                                          住所  \
0  コミュニティクリニックみさき「病児保育室わかば」     〒274-0812　船橋市三咲3-1-15 (コミュニティクリニックみさき隣)   
1      トレポンテこどもクリニック「トレポンテ」   〒273-0867千葉県船橋市夏見町2-566 (トレポンテこどもクリニック2階)   
2  さくら保育園「アイリスルーム」（病後児保育のみ）            〒273-0034千葉県船橋市二子町474 ( さくら保育園内)   
3              佐野医院「オー・キッズ」             〒273-0005千葉県船橋市本町3-3-7 (佐野医院2階)   
4         愛育レディースクリニック「アトム」  〒274-0071千葉県船橋市習志野5-10-30 （愛育レディースクリニック併設）   

        電話番号(代表)                                           geocodes  
0   047-448-4008  {'northeast': {'lat': 35.75123128029149, 'lng'...  
1   047-429-5403  {'northeast': {'lat': 35.7224354302915, 'lng':...  
2   047-302-8303  {'northeast': {'lat': 35.7110147802915, 'lng':...  
3  080-3416-0122  {'northeast': {'lat': 35.6987208302915, 'lng':...  
4  080-2151-1103  {'northeast': {'lat': 35.6990607802915, 'lng':...

児童ホーム
1


施設名                          住所      電話番号(代表)  \
0     前原児童ホーム    〒274-0825千葉県船橋市前原西6-1-12  047-475-5454   
1    高根台児童ホーム     〒274-0065千葉県船橋市高根台2-2-2  047-466-3638   
2   習志野台児童ホーム    〒274-0063千葉県船橋市習志野台3-4-1  047-464-0001   
3    金杉台児童ホーム     〒273-0852千葉県船橋市金杉台2-2-4  047-447-7837   
4     若松児童ホーム      〒273-0013千葉県船橋市若松2-3-6  047-433-7991   
5     西船児童ホーム       〒273-0033千葉県船橋市本郷町554  047-333-2007   
6     小室児童ホーム      〒270-1471千葉県船橋市小室町3308  047-457-1000   
7     三山児童ホーム     〒274-0072千葉県船橋市三山2-42-3  047-478-6800   
8   八木が谷児童ホーム    〒274-0804千葉県船橋市みやぎ台1-7-1  047-448-4400   
9    松が丘児童ホーム   〒274-0064千葉県船橋市松が丘1-52-22  047-463-5087   
10   飯山満児童ホーム  〒274-0822千葉県船橋市飯山満町2-488-8  047-468-0411   
11    夏見児童ホーム    〒273-0865千葉県船橋市夏見4-39-15  047-423-5999   
12    塚田児童ホーム    〒273-0042千葉県船橋市前貝塚町601-1  047-439-9299   
13    宮本児童ホーム     〒273-0003千葉県船橋市宮本6-18-1  047-424-9842   
14    三咲児童ホーム     〒274-0812千葉県船橋市三咲3-5-10  047-448-2397   
15   新高根児童ホーム    〒274-0814千葉県船橋市新高根1-12-9  047-469-3940   
16   薬円台児童ホーム    〒274-0077千葉県船橋市薬円台5-18-1  047-469-4884   
17    海神児童ホーム   〒273-0022千葉県船橋市海神町2-264-5  047-432-4661   
18    法典児童ホーム     〒273-0047千葉県船橋市藤原3-2-15  047-429-0813   
19   本中山児童ホーム     〒273-0035千葉県船橋市本中山1-6-6  047-333-7701   
20    坪井児童ホーム     〒274-0062千葉県船橋市坪井町755-4  047-468-1505   

                                             geocodes  
0   {'northeast': {'lat': 35.7040551802915, 'lng':...  
1   {'northeast': {'lat': 35.7356971, 'lng': 140.0...  
2   {'northeast': {'lat': 35.7232664302915, 'lng':...  
3   {'northeast': {'lat': 35.73725478029149, 'lng'...  
4   {'northeast': {'lat': 35.6806119802915, 'lng':...  
5   {'northeast': {'lat': 35.70961018029149, 'lng'...  
6   {'northeast': {'lat': 35.78942078029149, 'lng'...  
7   {'northeast': {'lat': 35.6977265802915, 'lng':...  
8   {'northeast': {'lat': 35.75772188029149, 'lng'...  
9   {'northeast': {'lat': 35.7373523802915, 'lng':...  
10  {'northeast': {'lat': 35.7092488802915, 'lng':...  
11  {'northeast': {'lat': 35.7181062802915, 'lng':...  
12  {'northeast': {'lat': 35.7248381802915, 'lng':...  
13  {'northeast': {'lat': 35.6990529802915, 'lng':...  
14  {'northeast': {'lat': 35.75385578029151, 'lng'...  
15  {'northeast': {'lat': 35.72529288029151, 'lng'...  
16  {'northeast': {'lat': 35.7115647802915, 'lng':...  
17  {'northeast': {'lat': 35.70246098029149, 'lng'...  
18  {'northeast': {'lat': 35.73622178029149, 'lng'...  
19  {'northeast': {'lat': 35.7169019802915, 'lng':...  
20  {'northeast': {'lat': 35.7339602802915, 'lng':...

子育て支援センター
1


施設名                       住所                         電話番号(代表)  \
0  南本町子育て支援センター   〒273-0004千葉県船橋市南本町10-1  047-434-3910・【相談専用】047-435-8333   
1  高根台子育て支援センター  〒274-0065千葉県船橋市高根台2-1-1  047-466-5666・【相談専用】047-466-3633   

                                            geocodes  
0  {'northeast': {'lat': 35.69850168029149, 'lng'...  
1  {'northeast': {'lat': 35.7362657802915, 'lng':...

その他預け先
0


以下の座標データを元に、正しくコーディングできているか、ランダムにテストする

In [95]:
for k in all_facilities.keys():
    print("Saving as csv:", k)
    print(len(all_facilities[k]))
    if len(all_facilities[k]) == 0:
        continue
    display(all_facilities[k][0]["geocodes"].astype(str).value_counts())
#     display(all_facilities[k][0])

Saving as csv: 保育園
1


{'northeast': {'lat': 35.7035542, 'lng': 140.0112155}, 'southwest': {'lat': 35.6894305, 'lng': 139.9974705}}                              2
{'northeast': {'lat': 35.71918618029149, 'lng': 140.0208115802915}, 'southwest': {'lat': 35.71648821970849, 'lng': 140.0181136197085}}    1
{'northeast': {'lat': 35.7152334802915, 'lng': 139.9716631802915}, 'southwest': {'lat': 35.7125355197085, 'lng': 139.9689652197085}}      1
{'northeast': {'lat': 35.7367117802915, 'lng': 140.0414424802915}, 'southwest': {'lat': 35.7340138197085, 'lng': 140.0387445197085}}      1
{'northeast': {'lat': 35.7550585802915, 'lng': 140.0377016802915}, 'southwest': {'lat': 35.7523606197085, 'lng': 140.0350037197085}}      1
                                                                                                                                         ..
{'northeast': {'lat': 35.74014028029149, 'lng': 139.9807984802915}, 'southwest': {'lat': 35.73744231970849, 'lng': 139.9781005197085}}    1
{'northeast': {'lat'

Saving as csv: 幼稚園
1


{'northeast': {'lat': 35.7227269802915, 'lng': 139.9978477802915}, 'southwest': {'lat': 35.72002901970851, 'lng': 139.9951498197085}}     1
{'northeast': {'lat': 35.75582728029149, 'lng': 140.0387533802915}, 'southwest': {'lat': 35.75312931970849, 'lng': 140.0360554197085}}    1
{'northeast': {'lat': 35.7296585802915, 'lng': 140.0325641802915}, 'southwest': {'lat': 35.7269606197085, 'lng': 140.0298662197085}}      1
{'northeast': {'lat': 35.7122926802915, 'lng': 139.9946850802915}, 'southwest': {'lat': 35.70959471970851, 'lng': 139.9919871197085}}     1
{'northeast': {'lat': 35.74316708029149, 'lng': 140.0412835802915}, 'southwest': {'lat': 35.74046911970849, 'lng': 140.0385856197085}}    1
{'northeast': {'lat': 35.70478488029149, 'lng': 140.0556099802915}, 'southwest': {'lat': 35.70208691970849, 'lng': 140.0529120197085}}    1
{'northeast': {'lat': 35.73333128029149, 'lng': 140.0137282802915}, 'southwest': {'lat': 35.7306333197085, 'lng': 140.0110303197085}}     1
{'northeast': {'lat'

Saving as csv: 認定こども園
1


{'northeast': {'lat': 35.73188278029149, 'lng': 139.9941549802915}, 'southwest': {'lat': 35.72918481970849, 'lng': 139.9914570197085}}    1
{'northeast': {'lat': 35.71241698029149, 'lng': 140.0226841802915}, 'southwest': {'lat': 35.7097190197085, 'lng': 140.0199862197085}}     1
{'northeast': {'lat': 35.7147413802915, 'lng': 140.0476365802915}, 'southwest': {'lat': 35.7120434197085, 'lng': 140.0449386197085}}      1
{'northeast': {'lat': 35.75638238029151, 'lng': 140.0262647802915}, 'southwest': {'lat': 35.75368441970851, 'lng': 140.0235668197085}}    1
{'northeast': {'lat': 35.7046346802915, 'lng': 140.0438490802915}, 'southwest': {'lat': 35.7019367197085, 'lng': 140.0411511197085}}      1
{'northeast': {'lat': 35.70173928029149, 'lng': 139.9851698802915}, 'southwest': {'lat': 35.6990413197085, 'lng': 139.9824719197085}}     1
{'northeast': {'lat': 35.7329678802915, 'lng': 140.0427829802915}, 'southwest': {'lat': 35.7302699197085, 'lng': 140.0400850197085}}      1
{'northeast': {'lat'

Saving as csv: 小規模保育事業所
1


{'northeast': {'lat': 35.6971234802915, 'lng': 139.9853266302915}, 'southwest': {'lat': 35.6944255197085, 'lng': 139.9826286697085}}      1
{'northeast': {'lat': 35.7128216802915, 'lng': 139.9813677802915}, 'southwest': {'lat': 35.71012371970851, 'lng': 139.9786698197085}}     1
{'northeast': {'lat': 35.7032507802915, 'lng': 139.9900152802915}, 'southwest': {'lat': 35.7005528197085, 'lng': 139.9873173197085}}      1
{'northeast': {'lat': 35.7020278802915, 'lng': 140.0332515802915}, 'southwest': {'lat': 35.6993299197085, 'lng': 140.0305536197085}}      1
{'northeast': {'lat': 35.72445648029149, 'lng': 140.0416347302915}, 'southwest': {'lat': 35.72175851970849, 'lng': 140.0389367697085}}    1
{'northeast': {'lat': 35.7167684802915, 'lng': 140.0455034802915}, 'southwest': {'lat': 35.7140705197085, 'lng': 140.0428055197085}}      1
{'northeast': {'lat': 35.70086798029149, 'lng': 140.0064928802915}, 'southwest': {'lat': 35.69817001970849, 'lng': 140.0037949197085}}    1
{'northeast': {'lat'

Saving as csv: 家庭的保育者
1


{'northeast': {'lat': 35.7175555, 'lng': 139.9764377}, 'southwest': {'lat': 35.711041, 'lng': 139.9709179}}     1
{'northeast': {'lat': 35.7375105, 'lng': 140.0550643}, 'southwest': {'lat': 35.7322745, 'lng': 140.0477109}}    1
{'northeast': {'lat': 35.7022897, 'lng': 140.0519363}, 'southwest': {'lat': 35.6986704, 'lng': 140.0460579}}    1
{'northeast': {'lat': 35.7278249, 'lng': 140.0240107}, 'southwest': {'lat': 35.720676, 'lng': 140.017224}}      1
{'northeast': {'lat': 35.7006683, 'lng': 140.0467409}, 'southwest': {'lat': 35.6948039, 'lng': 140.0383092}}    1
Name: geocodes, dtype: int64

Saving as csv: 事業所内保育事業所
0
Saving as csv: 認証保育所
1


{'northeast': {'lat': 35.69606178029151, 'lng': 139.9932237802915}, 'southwest': {'lat': 35.69336381970851, 'lng': 139.9905258197085}}    1
{'northeast': {'lat': 35.72264098029149, 'lng': 140.0435469802915}, 'southwest': {'lat': 35.7199430197085, 'lng': 140.0408490197085}}     1
{'northeast': {'lat': 35.7066708302915, 'lng': 139.9869966802915}, 'southwest': {'lat': 35.7039728697085, 'lng': 139.9842987197085}}      1
Name: geocodes, dtype: int64

Saving as csv: 認可外保育施設
1


{'northeast': {'lat': 35.7060489802915, 'lng': 139.9878117802915}, 'southwest': {'lat': 35.7033510197085, 'lng': 139.9851138197085}}      1
{'northeast': {'lat': 35.7085701302915, 'lng': 139.9593834802915}, 'southwest': {'lat': 35.7058721697085, 'lng': 139.9566855197085}}      1
{'northeast': {'lat': 35.7060324802915, 'lng': 140.0582266802915}, 'southwest': {'lat': 35.7033345197085, 'lng': 140.0555287197085}}      1
{'northeast': {'lat': 35.6888507802915, 'lng': 139.9946140802915}, 'southwest': {'lat': 35.6861528197085, 'lng': 139.9919161197085}}      1
{'northeast': {'lat': 35.68615208029149, 'lng': 139.9935964802915}, 'southwest': {'lat': 35.68345411970849, 'lng': 139.9908985197085}}    1
{'northeast': {'lat': 35.7173542802915, 'lng': 139.9866254302915}, 'southwest': {'lat': 35.7146563197085, 'lng': 139.9839274697085}}      1
{'northeast': {'lat': 35.7079285802915, 'lng': 140.0266474302915}, 'southwest': {'lat': 35.7052306197085, 'lng': 140.0239494697085}}      1
{'northeast': {'lat'

Saving as csv: 一時保育
1


{'northeast': {'lat': 35.7620307802915, 'lng': 140.0342261802915}, 'southwest': {'lat': 35.7593328197085, 'lng': 140.0315282197085}}      1
{'northeast': {'lat': 35.7339109802915, 'lng': 139.9758316802915}, 'southwest': {'lat': 35.7312130197085, 'lng': 139.9731337197085}}      1
{'northeast': {'lat': 35.74771468029149, 'lng': 139.9954124302915}, 'southwest': {'lat': 35.7450167197085, 'lng': 139.9927144697085}}     1
{'northeast': {'lat': 35.75582728029149, 'lng': 140.0387533802915}, 'southwest': {'lat': 35.75312931970849, 'lng': 140.0360554197085}}    1
{'northeast': {'lat': 35.76638003029149, 'lng': 140.0569219302915}, 'southwest': {'lat': 35.76368206970849, 'lng': 140.0542239697085}}    1
{'northeast': {'lat': 35.7046346802915, 'lng': 140.0438490802915}, 'southwest': {'lat': 35.7019367197085, 'lng': 140.0411511197085}}      1
{'northeast': {'lat': 35.6799572802915, 'lng': 139.9978598802915}, 'southwest': {'lat': 35.6772593197085, 'lng': 139.9951619197085}}      1
{'northeast': {'lat'

Saving as csv: 病児・病後児保育
1


{'northeast': {'lat': 35.6990607802915, 'lng': 140.0634075802915}, 'southwest': {'lat': 35.6963628197085, 'lng': 140.0607096197085}}      1
{'northeast': {'lat': 35.75123128029149, 'lng': 140.0327135802915}, 'southwest': {'lat': 35.74853331970849, 'lng': 140.0300156197085}}    1
{'northeast': {'lat': 35.6987208302915, 'lng': 139.9889058802915}, 'southwest': {'lat': 35.6960228697085, 'lng': 139.9862079197085}}      1
{'northeast': {'lat': 35.7110147802915, 'lng': 139.9515132802915}, 'southwest': {'lat': 35.7083168197085, 'lng': 139.9488153197085}}      1
{'northeast': {'lat': 35.7224354302915, 'lng': 139.9985738302915}, 'southwest': {'lat': 35.71973746970851, 'lng': 139.9958758697085}}     1
Name: geocodes, dtype: int64

Saving as csv: 児童ホーム
1


{'northeast': {'lat': 35.73725478029149, 'lng': 140.0138270802915}, 'southwest': {'lat': 35.73455681970849, 'lng': 140.0111291197085}}    1
{'northeast': {'lat': 35.7339602802915, 'lng': 140.0596460802915}, 'southwest': {'lat': 35.7312623197085, 'lng': 140.0569481197085}}      1
{'northeast': {'lat': 35.72529288029151, 'lng': 140.0230649802915}, 'southwest': {'lat': 35.72259491970851, 'lng': 140.0203670197085}}    1
{'northeast': {'lat': 35.7232664302915, 'lng': 140.0500847302915}, 'southwest': {'lat': 35.7205684697085, 'lng': 140.0473867697085}}      1
{'northeast': {'lat': 35.70246098029149, 'lng': 139.9763506802915}, 'southwest': {'lat': 35.6997630197085, 'lng': 139.9736527197085}}     1
{'northeast': {'lat': 35.6990529802915, 'lng': 139.9963831802915}, 'southwest': {'lat': 35.69635501970851, 'lng': 139.9936852197085}}     1
{'northeast': {'lat': 35.6977265802915, 'lng': 140.0507410802915}, 'southwest': {'lat': 35.6950286197085, 'lng': 140.0480431197085}}      1
{'northeast': {'lat'

Saving as csv: 子育て支援センター
1


{'northeast': {'lat': 35.69850168029149, 'lng': 139.9785855802915}, 'southwest': {'lat': 35.69580371970849, 'lng': 139.9758876197085}}    1
{'northeast': {'lat': 35.7362657802915, 'lng': 140.0300387802915}, 'southwest': {'lat': 35.7335678197085, 'lng': 140.0273408197085}}      1
Name: geocodes, dtype: int64

Saving as csv: その他預け先
0


- エラーなくデータが取得できている？

- 保育園の住所に重複が見られる
    - `{'northeast': {'lat': 35.7035542, 'lng': 140.0112155}, 'southwest': {'lat': 35.6894305, 'lng': 139.9974705}}`

In [96]:
for k in all_facilities.keys():
    print("Saving as csv:", k)
    print(len(all_facilities[k]))
    if len(all_facilities[k]) == 0:
        continue
    all_facilities[k][0].to_csv(f"{k}.csv", index=False)
#     display(all_facilities[k][0])

Saving as csv: 保育園
1
Saving as csv: 幼稚園
1
Saving as csv: 認定こども園
1
Saving as csv: 小規模保育事業所
1
Saving as csv: 家庭的保育者
1
Saving as csv: 事業所内保育事業所
0
Saving as csv: 認証保育所
1
Saving as csv: 認可外保育施設
1
Saving as csv: 一時保育
1
Saving as csv: 病児・病後児保育
1
Saving as csv: 児童ホーム
1
Saving as csv: 子育て支援センター
1
Saving as csv: その他預け先
0


In [106]:
all_facilities["保育園"][0][all_facilities["保育園"][0]["geocodes"].astype(str).duplicated(keep=False)]

施設名                       住所                          電話番号(代表)  \
82  東船橋ちとせ保育園  〒273-0002　船橋市東船橋1-16-25  電話：047-423-5717 FAX：047-423-5718   
93   グリュック保育園  〒273-0002　船橋市東船橋1-26-10  電話：047-424-3741 FAX:047-455-3457   

                                             geocodes  
82  {'northeast': {'lat': 35.7035542, 'lng': 140.0...  
93  {'northeast': {'lat': 35.7035542, 'lng': 140.0...

この2件だけ同じgeocodingしていました。

なんでや...?